sites/main-site/src/content/blog/2025/switching-master-to-main.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: "nf-core goes woke!"
+subtitle: "Announcing the possibility to change a pipeline default branch to main"
+pubDate: 2025-05-14T12:00:00+01:00
```

Comment:
don't forget to update this before merging

Reply:
```suggestion
pubDate: 2025-05-15T12:00:00+01:00
```